ludwig/utils/backward_compatibility.py

```diff
@@ -186,24 +216,35 @@ def _upgrade_preprocessing(preprocessing: Dict[str, Any]):
             DeprecationWarning,
         )
 
-        if "type" not in split_params:
-            split_params["type"] = "random"
+        if TYPE not in split_params:
+            split_params[TYPE] = RANDOM
 
     if split_params:
-        preprocessing["split"] = split_params
+        preprocessing[SPLIT] = split_params
+
+    if AUDIO in preprocessing:
+        if "audio_feature" in preprocessing[AUDIO]:
+            for k, v in preprocessing[AUDIO]["audio_feature"].items():
+                preprocessing[AUDIO][k] = v
+            del preprocessing[AUDIO]["audio_feature"]
+        warnings.warn(
+            "Parameters specified at the `audio_feature` parameter level have been unnested and should now "
+            "be specified at the preprocessing level",
```

Comment:
Please mention the removal version to be v0.7.

Reply:
Will do!